src/tests/Common/GenerateHWIntrinsicTests/GenerateHWIntrinsicTests_Arm.cs

```diff
@@ -150,6 +159,7 @@
     ("_SveMinimalUnaryOpTestTemplate.template", "SveVecReduceUnOpTest.template",  new Dictionary<string, string> { ["TemplateName"] = "Simple",      ["TemplateValidationLogic"] = VecReduceOpTest_ValidationLogic }),
     ("_SveMasklessUnaryOpTestTemplate.template","SveMasklessSimpleVecOpTest.template", new Dictionary<string, string> { ["TemplateName"] = "Simple",      ["TemplateValidationLogic"] = SimpleVecOpTest_ValidationLogic }),
     ("_SveStoreTemplate.template",              "SveStoreAndZipTest.template",    new Dictionary<string, string> { ["TemplateName"] = "Simple",      ["TemplateValidationLogic"] = SimpleVecOpTest_ValidationLogic }),
+    ("_SveStoreTemplate.template",              "SveStoreNarrowingTest.template", new Dictionary<string, string> { ["TemplateName"] = "Simple",      ["TemplateValidationLogic"] = SimpleVecOpTest_ValidationLogicForNarrowing }),
     ("_SveStoreTemplate.template",              "SveStoreNonTemporalTest.template",new Dictionary<string, string> { ["TemplateName"] = "Simple",     ["TemplateValidationLogic"] = SimpleVecOpTest_ValidationLogic }),
```

Comment:
`SveStoreNonTemporalTest.template` and `SveStoreAndZipTest.template` is identical too. Can we just have one?

Reply:
Done.